config/core/test-configs.yaml

```diff
@@ -584,6 +584,11 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  de0-nano-soc:
+    mach: socfpga
+    class: arm-dtb
+    boot_method: uboot
```

Comment:
```suggestion
    boot_method: uboot
    dtb: 'socfpga_cyclone5_de0_nano_soc.dtb'
```

Reply:
The dtb name doesn't match the device type name, so this extra property is needed.

Reply:
thanks
updating this

Reply:
It looks like this device type is not in the CIP lab:
https://lava.ciplatform.org/scheduler/device_types

Reply:
here it is https://lava.ciplatform.org/scheduler/device_type/de0-nano-soc

Reply:
Ah yes sorry, got misled again because of the different dtb name.

However, it's still not getting any KernelCI jobs from staging for some reason.